Tests/RxSwiftTests/PrimitiveSequence+DebugTest.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PrimitiveSequence+DebugTest.swift
+//  Rx
+//
+//  Created by muukii on 3/13/17.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxTest
+
+class PrimitiveSequenceDebugTest : RxTest {
+
+}
+
+// MARK: debug
+extension PrimitiveSequenceDebugTest {
+  func testDebug_producesSingleElement() {
```

Comment:
Could you please move this tests to `PrimitiveSequenceTest` and group them with other tests.